### ISSUE TITLE
Suggestion: Don't warn about unused import aliases in some edge cases

### ISSUE THREAD

**Issue by matheus23** [stil4m/elm-analyse#234]
Hi! I'm loving `elm-analyse` so far :)

In one of my files I have these imports:
```elm
import Svg as UntypedSvg exposing (Svg)
import TypedSvg as Svg
import TypedSvg.Types as Svg
```

I don't use the `UntypedSvg` alias, but if I were to remove it, the `Svg` import would clash with my aliased `import TypedSvg as Svg`.

Maybe my work-around for imports is questionable. What do you think is the best way to resolve this? Should the exist an option for allowing what I'm doing, or should everyone never alias to a name that's already imported?

**Comment by matheus23** [comment]
I have since revised what I think of this rule. It _should_ warn that something is unused, when something is unused, period. Even if one needed to add an import so that something else doesn't break.

And for anyone wondering, there's a type alias to `Svg` in `TypedSvg.Core` ;)
